cogboard-webapp/src/components/widgets/types/LogViewerWidget/index.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import { number, string } from 'prop-types';
+
+const LogViewerWidget = ({
+  id,
+  endpoint,
+  schedulePeriod,
+  path,
+  logLinesField,
+  logFileSizeField,
+  logRecordExpirationField
+}) => {
+  return (
+    <>
+      id: {id}
+      <br />
```

Comment:
I think it will be better if you use CSS/Styled component to imitate break line instead of using <br> tag. If you need any information just ping me on priv

Reply:
It is a placeholder for the widget UI. It is not required so I will remove it.